src/components/CorrelationTable.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div id="correlations" v-if="rows > 0">
+        <div class="text-right mb-2">
+            <label class="label">
+                <input type="checkbox" v-model="sortByCorrelation"/> Sort by correlation
+            </label>
+            <csv-download
+                :data="correlationData"
+                filename="genetic_correlations"
+            ></csv-download>
+        </div>
+        <b-table
+            hover
+            small
+            responsive="sm"
+            :items="tableData"
+            :fields="fields"
+            :per-page="perPage"
+            :current-page="currentPage"
+        >
+            <template v-slot:cell(link)="r">
+                <a :href="`/phenotype.html?phenotype=${r.item['other_phenotype']}`">{{
+                            getDescription(r.item['other_phenotype'])}}
+                </a>
+            </template>
+        </b-table>
+        <b-pagination
+            class="pagination-sm justify-content-center"
+            v-model="currentPage"
+            :total-rows="rows"
+            :per-page="perPage"
+        ></b-pagination>
+    </div>
+</template>
+<script>
+import Vue from "vue";
+import { orderBy, groupBy } from "lodash";
+import Formatters from "@/utils/formatters";
+import uiUtils from "@/utils/uiUtils";
+import { decodeNamespace } from "@/utils/filterHelpers";
```

Comment:
Is this function being used anywhere in the component?

Reply:
You mean decodeNamespace? No, I think I was trying to replicate another component exactly and realized I didn't need some of those. I need formatters but I don't think I need the rest.